vm/systemSmartContracts/validator.go

```diff
@@ -1127,12 +1160,19 @@ func (v *validatorSC) activateStakingFor(
 		}
 
 		if stakedData.UnStakedNonce == 0 {
-			numRegistered++
+			numActivatedKey++
+		}
+
+		nodesActivated++
```

Comment:
Is it ok that you can increase two times the num of activated nodes here?

Reply:
those are 2 different variables.